l2-contracts/contracts/bridge/L2ERC20Bridge.sol

```diff
@@ -28,6 +28,8 @@ contract L2ERC20Bridge is IL2Bridge, Initializable {
     /// @dev Bytecode hash of the proxy for tokens deployed by the bridge.
     bytes32 internal l2TokenProxyBytecodeHash;
 
+    address public governor;
```

Comment:
Since Era should be upgraded too, you must add this variable at the end, otherwise storage layout will messed up.

Reply:
Yes, I will make it Ownable2StepRandomAddres Instead